autoload.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
-require __DIR__.'/src/functions_include.php';
+require __DIR__ . '/src/functions_include.php';
 
-spl_autoload_register(function ($class) {
+spl_autoload_register(static function ($class) {
```

Comment:
the addition of `static` is more of a micro-optimization than anything else I would not add it here

Reply:
Okay, I see.